tests/integration/components/basic-dropdown/content-test.js

```diff
@@ -103,12 +103,54 @@ module('Integration | Component | basic-dropdown/content', function(hooks) {
     await render(hbs`
       <div id="destination-el"></div>
       <div id="other-div"></div>
-      {{#basic-dropdown/content dropdown=dropdown destination='destination-el'}}Lorem ipsum{{/basic-dropdown/content}}
+      {{#basic-dropdown/content rootEventType='mousedown' dropdown=dropdown destination='destination-el'}}Lorem ipsum{{/basic-dropdown/content}}
```

Comment:
@cibernox I added the `rootEventType` here because the default is defined in the `basic-dropdown` component. Is this the correct pattern or should the fallback be moved down into the content? Content shouldn't really exist without the parent component binding, so I figure this is ok, but I'm not sure whether to add this for the remaining tests to ensure click behaviour isn't unexpected.